app/controllers/dashboard_controller.rb

```diff
@@ -2,8 +2,8 @@ class DashboardController < ApplicationController
   include Foreman::Controller::AutoCompleteSearch
   include Foreman::Controller::Parameters::Widget
 
-  before_action :prefetch_data, :only => :index
-  before_action :find_resource, :only => [:destroy]
+  before_action :prefetch_data, :only => :show
```

Comment:
Long term we can move the common widgets to be triggered via JavaScript with the same data. If this was done in react, you would have two components using the same store data ( where flux would fetch it only once)